### PR TITLE
Forces and Torques (Loads)

### PR DESCRIPTION
**Pull request by moorepants**

### CODE REVIEW

File: loads.rst [hunk 1]
```diff
@@ -0,0 +1,593 @@
...
+   me.init_vprinting(use_latex='mathjax')
+
+Force
+=====
+
+A *force* is an abstraction we use to describe the thing that causes mass to
+move (i.e. accelerate from a stationary state). There are four `fundamental
+forces of nature`_ of which all other forces can be derived from. Moments and
+torques can be described using forces and have analogous relationships as
+particles have to a rigid body being made up of a set of particles spatially
```
Comment (jtheinen) [inline loads.rst:24]: Not sure what you mean with the analogous relationships as particles have to a rigid body being made up of a set of particles spatially fixed together.

File: loads.rst [hunk 2]
```diff
@@ -0,0 +1,593 @@
...
+fixed together. Forces, moments, and torques have magnitude and direction and
+thus we use vectors to describe them mathematically.
+
+.. _fundamental forces of nature: https://en.wikipedia.org/wiki/Force#Fundamental_forces
```
Comment (jtheinen) [inline loads.rst:28]: Link not working in preview

File: loads.rst [hunk 3]
```diff
@@ -0,0 +1,593 @@
...
+If each vector in the resultant is bound, the sum of the moments due to each
+vector about :math:`P` is call the moment of :math:`\bar{R}^{S}` about
+:math:`P`.  This summation can be written as:
+
+.. math::
+   :label: eq-sum-moments
+
+   \bar{M}^{S/P} = \sum_{i=1}^{\nu} \bar{r}^{L_i/P} \times \bar{v}_i
+
+The moment of bound vectors :math:`S` about one point :math:`P` is related to
```
Comment (jtheinen) [inline loads.rst:77]: line 77 untill 86 is not completely clear for me. R_S and R_SQ are different vectors, how would you know the vector R_SQ of an arbitrary point Q? By projecting R_S on a line that goes through Q?

File: loads.rst [hunk 4]
```diff
@@ -0,0 +1,593 @@
...
+   FL.orient_axis(W, alphafl, W.z)
+   RR.orient_axis(B, alpharr, B.z)
+   RL.orient_axis(B, alpharl, B.z)
+
+The resultant of the forces expressed in the :math:`B` frame is then:
+
+.. jupyter-execute::
+
+   R = Ffl*FL.x + Ffr*FR.x + Frl*RL.x + Frr*RR.x
+   R.express(B).simplify()
```
Comment (jtheinen) [inline loads.rst:192]: Output does not fit my screen

File: loads.rst [hunk 5]
```diff
@@ -0,0 +1,593 @@
...
+*Non-contributing forces* do no work on the system. For example, when a force
+acts between two points that have no relative motion, no work is done. Examples
+of non-contributing forces:
+
+a. contact forces on particles across smooth (frictionless) surfaces of rigid
+   bodies
+b. any internal contact and body (distance) forces between any two points in a
+   rigid body
+c. contact forces between bodies rolling without slipping on each other (special
+   case of a.)
```
Comment (jtheinen) [inline loads.rst:327]: In preview the examples are listed 1 2 and 3

File: loads.rst [hunk 6]
```diff
@@ -0,0 +1,593 @@
...
+
+Later we will see how generalized coordinates relieve us from having to specify
+an non-contributing forces.
```
Comment (jtheinen) [inline loads.rst:330]: remove an

File: loads.rst [hunk 7]
```diff
@@ -0,0 +1,593 @@
...
+
+Gravity
+=======
+
+We will often be interested in a multibody systems motion when it is subject to
+gravitational forces. The simplest case is a constant unidirectional
+gravitational field, which is appropriate model for objects moving on and near
```
Comment (jtheinen) [inline loads.rst:337]: which is "an" appropriate model

File: loads.rst [hunk 8]
```diff
@@ -0,0 +1,593 @@
...
+the Earth's surface. The gravitational forces can be applied solely to the mass
+centers of each rigid body in a multibody system as a resultant force. The
+gravitational torque on the bodies is zero because the force is equal in
+magnitude for each particle in the body. See [Kane1985]_ pg. 110 for the more
+general case of `Newton's Law of Universal Gravitation`_ where this is not the
```
Comment (jtheinen) [inline loads.rst:342]: Change case to avoid double use of case in one sentence

File: loads.rst [hunk 9]
```diff
@@ -0,0 +1,593 @@
...
+   Fg = -m*g*N.y
+   Fg
+
+Springs & Dampers
+=================
+
+Idealized springs and dampers are useful models of elements that have distance
+and velocity dependent forces and torques. A spring with free length
+:math:`q_0` and where :math:`q_1,q_2` locate the ends of the spring along a
+line parallel to the :math:`\hat{n}_x`.
```
Comment (jtheinen) [inline loads.rst:364]: add axis or remove the

File: loads.rst [hunk 10]
```diff
@@ -0,0 +1,593 @@
...
+
+If we displace :math:`P` in the positive :math:`\hat{n}_x` direction the spring
```
Comment (jtheinen) [inline loads.rst:366]: Point P is not described or shown

File: loads.rst [hunk 11]
```diff
@@ -0,0 +1,593 @@
+=========================
```
Comment (jtheinen) [inline loads.rst:1]: Document is good to read and good examples. The moment section seems a bit rough for me. The rest is very clear!

File: loads.rst [hunk 12]
```diff
@@ -0,0 +1,593 @@
...
+
+Here a positive displacement represents the spring in tension and a negative
+displacement is compression.
+
+.. jupyter-execute::
+
+   Fs = -k*displacement*N.x
+   Fs
+
+Similarly, a linear damping force with damping coefficient :math:`c` is defined
```
Comment (jtheinen) [inline loads.rst:396]: Maybe elaborate a bit more on damping

File: loads.rst (hunk 1 above)
Comment (moorepants) [reply loads.rst:24]: Remove this sentence and swapped it with some simpler statements.

File: loads.rst (hunk 2 above)
Comment (moorepants) [reply loads.rst:28]: That might just be an issue with the preview. Works in my local build.

File: loads.rst (hunk 3 above)
Comment (moorepants) [reply loads.rst:77]: R_S and R_SQ aren't different vectors. It is only that you treat R_SQ as being a bound vector, bound to a line of action through Q. You don't know the vector at the point, you declare it is a vector bound to that point.

Comment (moorepants) [reply loads.rst:77]: I've added a simple example to show how this works.

File: loads.rst (hunk 4 above)
Comment (moorepants) [reply loads.rst:192]: This wraps in the actual build.

File: loads.rst (hunk 5 above)
Comment (moorepants) [reply loads.rst:327]: Changed to numbers.

File: loads.rst (hunk 6 above)
Comment (moorepants) [reply loads.rst:330]: fixed

File: loads.rst (hunk 7 above)
Comment (moorepants) [reply loads.rst:337]: fixed

File: loads.rst (hunk 8 above)
Comment (moorepants) [reply loads.rst:342]: fixed

File: loads.rst (hunk 9 above)
Comment (moorepants) [reply loads.rst:364]: fixed

File: loads.rst (hunk 10 above)
Comment (moorepants) [reply loads.rst:366]: added to the figure

File: loads.rst (hunk 12 above)
Comment (moorepants) [reply loads.rst:396]: fixed

**Pull request merged**